tests/integration/component_provider/python/recursive-types/provider/PulumiPlugin.yaml

```diff
@@ -0,0 +1,5 @@
+runtime:
+  name: python
+  options:
+    toolchain: pip
+    virtualenv: venv
```

Comment:
Do we need options here?

Reply:
No, good call out. These are the defaults that are used anyway, but we tell users to not set options, so we should not do that in our tests either.